app/authentication/jti_claim_storage.py

```diff
@@ -41,10 +45,12 @@ def use_jti_claim(jti_claim):
         raise TypeError
 
     try:
-        jti = UsedJtiClaim(jti_claim)
-        # pylint: disable=maybe-no-member
-        db.session.add(jti)
-        db.session.commit()
-    except IntegrityError as e:
+        used_at = datetime.now(tz=tzutc())
+        valid_until = used_at + timedelta(seconds=current_app.config['EQ_USED_JTI_CLAIM_TTL_SECONDS'])
```

Comment:
I think we should use the `exp` value in the JWT for this.
http://ons-schema-definitions.readthedocs.io/en/latest/respondent_management_to_electronic_questionnaire.html